src/Actions/BcgovSettings.php

```diff
@@ -285,6 +318,49 @@ public function bcgov_external_link_icons_settings() {
 		<?php
     }
 
+	/**
+     * Sets the Google Site Name.
```

Comment:
don't think this comment matches the function

Reply:
Fixed.